README.md

```diff
@@ -3,7 +3,7 @@
 
 # Morganey [![Status Ventis](https://img.shields.io/badge/status-ventis-yellow.svg)](https://github.com/ForNeVeR/andivionian-status-classifier)
 
-[Lambda Calculus][wiki-lambda-calculus] based programming language
+[Lambda Calculus][wiki-lambda-calculus] interpreted based programming language
```

Comment:
It sounds for me you'd like to enumerate the features and properties of Morganey. I think for that you'd rather say

> [Lambda Calculus][wiki-lambda-calculus] based, interpreted programming language that recognizes Church encoded structures.

instead of

> [Lambda Calculus][wiki-lambda-calculus] interpreted based programming language that recognizes Church encoded structures.